### PR TITLE
fix: accesibility

### LINKED ISSUE #254: Fix accessibility warnings
In the Google Play Pre-launch report, we accessibility several warnings :
- 2 warnings related to Content labelling.
- 1 warning related to the touch target size.

To be able to reproduce these errors, we can use an [accessibility scanner](https://docs.flutter.dev/accessibility-and-localization/accessibility?tab=talkback#inspecting-accessibility-support)

### PR DESCRIPTION
**Pull request by gabaldon**
- [x] Add semantics to non labeled widgets.
- [x] Add new button types for box-button and icon-button
- [x] Update focus style.

close #254

### CODE REVIEW

File: lib/widgets/layouts/dashboard_layout.dart [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import 'dart:async';
 
 import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
```
Comment (Tommytrg) [inline lib/widgets/layouts/dashboard_layout.dart:4]:
```suggestion
```

File: lib/widgets/layouts/layout.dart [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import 'dart:async';
 import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
```
Comment (Tommytrg) [inline lib/widgets/layouts/layout.dart:3]:
```suggestion
```

File: lib/theme/extended_theme.dart [hunk 3]
```diff
@@ -118,63 +122,66 @@ class ExtendedTheme extends ThemeExtension<ExtendedTheme> {
...
+        monoSmallText: monoSmallText ?? this.monoSmallText,
+        monoRegularText: monoRegularText ?? this.monoRegularText,
+        monoMediumText: monoMediumText ?? this.monoMediumText,
+        monoLargeText: monoLargeText ?? this.monoLargeText,
+        numberPaginatiorSelectedBg:
+            numberPaginatiorSelectedBg ?? this.numberPaginatiorSelectedBg,
+        numberPaginatiorUnselectedFg:
+            numberPaginatiorUnselectedFg ?? this.numberPaginatiorUnselectedFg,
+        darkBgFocusColor: darkBgFocusColor ?? this.darkBgFocusColor,
+        focusBg: focusBg ?? this.focusBg);
```
Comment (Tommytrg) [inline lib/theme/extended_theme.dart:184]: Are we only adding this two lines here?

File: lib/shared/api_database.dart [hunk 4]
```diff
@@ -70,6 +70,14 @@ class ApiDatabase {
...
     Map<String, dynamic> addressList = {'$currentWalletId': '0/0'};
 
+    // if currentWalletId exists, preferences should be re-written in local storage
+    if (currentWalletId != null) {
+      await setWalletAndAccountInLocalStorage(
+          currentWalletId,
+          AddressEntry(
+              walletId: currentWalletId, addressIdx: addressId, keyType: 0));
+    }
+
```
Comment (Tommytrg) [inline lib/shared/api_database.dart:80]: Could we remove this? It's already included in https://github.com/witnet/my-wit-wallet/pull/259
```suggestion
```

File: lib/shared/api_database.dart [hunk 5]
```diff
@@ -84,7 +92,6 @@ class ApiDatabase {
     ApiDatabase db = Locator.instance<ApiDatabase>();
     WalletStorage walletStorage = db.walletStorage;
     int addressIndex = int.parse(addressId.split('/').last);
-    int addressKeyType = int.parse(addressId.split('/').first);
```
Comment (Tommytrg) [inline lib/shared/api_database.dart:87]: Included in https://github.com/witnet/my-wit-wallet/pull/259

File: lib/shared/api_database.dart [hunk 6]
```diff
@@ -97,16 +104,6 @@ class ApiDatabase {
...
-
-      // if currentWalletId exists, preferences should be re-written in local storage
-      if (currentWalletId != null) {
-        setWalletAndAccountInLocalStorage(
-            currentWalletId,
-            AddressEntry(
-                walletId: currentWalletId,
-                addressIdx: accountPreferences[AccountPreferences.addressIndex],
-                keyType: addressKeyType));
-      }
```
Comment (Tommytrg) [inline lib/shared/api_database.dart:109]: Included in https://github.com/witnet/my-wit-wallet/pull/259

**Review by Tommytrg — CHANGES_REQUESTED** [review]

File: lib/main.dart [hunk 7]
```diff
@@ -19,7 +19,7 @@ void main() async {
   if (Platform.isWindows || Platform.isMacOS || Platform.isLinux) {
     await windowManager.ensureInitialized();
     WindowOptions windowOptions = WindowOptions(
-      size: Size(400, 660),
+      size: Size(400, 700),
```
[the remark this answers is not in the document]
Comment (gabaldon) [reply lib/main.dart:22]: The input error was not completely visible

File: lib/screens/create_wallet/enc_xprv_card.dart [hunk 8]
```diff
@@ -87,31 +98,38 @@ class EnterXprvCardState extends State<EnterEncryptedXprvCard>
...
-                                        setState(() {
-                                          _errorXprvText = _validateXprv(xprv);
-                                        })
-                                      })))
-                    },
-                color:
-                    theme.inputDecorationTheme.enabledBorder?.borderSide.color)
+            ? Semantics(
+                label: 'Scan QR code',
+                child: IconButton(
```
[the remark this answers is not in the document]
Comment (gabaldon) [reply lib/screens/create_wallet/enc_xprv_card.dart:107]: Yes, we don't want the default icon color, in this case, it is the sufix of the input and we need to define here the focused style.

**Pull request merged**